File: help/pt_pt/exercicios_da_ficha_11/Ficha11_015/Ficha11_015.py
# Programador           -


#::::::::::::::::::::::::::::::::::::::::::::::::::
#User Defined Functions Code
#::::::::::::::::::::::::::::::::::::::::::::::::::
def linha(n, c):
    for i in range(0, n, 1 ):
        print(c  , end="")
    print()

#::::::::::::::::::::::::::::::::::::::::::::::::::
def interior(n, cl, ci):
    print(cl  , end="")
    for i in range(1, n - 1, 1 ):
        print(ci  , end="")
    print(cl)

#::::::::::::::::::::::::::::::::::::::::::::::::::
def caixa(larg, alt, cl, ci):
    linha(larg , cl )
    for i in range(1, alt - 1, 1 ):
        interior(larg , cl , ci )
    linha(larg , cl )

if __name__ == "__main__":
    altura =  int(input("Altura   : "))
    largura =  int(input("Largura  : "))
    limite =  str(input("Limite   : "))
    interior =  str(input("Interior : "))
    caixa(largura , altura , limite , interior )


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	quarta, 2022 out 12 20:14

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_015/Ficha11_015.js
// Programador           -

//Main Function
function main() {
    document.write("Altura   : " );// print text
    altura = parseInt(prompt("Altura   : " ,"0") );
    
    document.write(altura + "\n");// print input 
    document.write("Largura  : " );// print text
    largura = parseInt(prompt("Largura  : " ,"0") );
    
    document.write(largura + "\n");// print input 
    document.write("Limite   : " );// print text
    limite = prompt("Limite   : " ,"");
    
    document.write(limite + "\n");// print input 
    document.write("Interior : " );// print text
    interior = prompt("Interior : " ,"");
    
    document.write(interior + "\n");// print input 
    caixa(largura , altura , limite , interior ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linha(n, c) {
    for( var i = 0; i < n; i = i + 1 ) {
        document.write(c );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function interior(n, cl, ci) {
    document.write(cl );
    for( var i = 1; i < n - 1; i = i + 1 ) {
        document.write(ci );
    }
    document.write(cl + "\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function caixa(larg, alt, cl, ci) {
    linha(larg , cl ) ;
    for( var i = 1; i < alt - 1; i = i + 1 ) {
        interior(larg , cl , ci ) ;
    }
    linha(larg , cl ) ;
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:14
